### DISCUSSION TITLE
Setting new timezone-utc-offset results in odd read back

### DISCUSSION THREAD

**Discussion by troglobit** [kernelkit/infix#176] [Q&A] [answered]
When playing with the recently added timezone-utc-offset in ietf-system, I ran into an odd issue when reading back the current-datetime from the system operational datastore.  This was even more odd when using timezone-name, because the client presents the remote system time with my local timezone.  Can anyone please explain this?  Btw, I'm using netopeer2-cli.

**Comment by mattiaswal** [comment, marked as answer]
The value is transferred correct (with timezone information)  from infix, but netopeer2-cli (libyang) is transforming it to a canonical date, which results in that the time is presented in your local timezone in netopper2-cli.

You can see more here: https://github.com/CESNET/libyang/issues/2116

**Reply by troglobit** [reply]
Thank you! I had to ask, was not apparent at all(!) from the YANG model.  I've tested now with another (Python) client and I can see the XML has the correct value I set before.
